src/main/java/fi/otavanopisto/kuntaapi/server/integrations/ptv/PtvTranslator.java

```diff
@@ -708,5 +688,44 @@ private ServiceChannelAttachment translateAttachment(fi.otavanopisto.restfulptv.
     return result;
   }
   
+  private List<LocalizedValue> mergeDescriptions(List<LocalizedValue> statutoryDescriptions, List<LocalizedValue> descriptions) {
+    if (statutoryDescriptions == null && descriptions == null) {
+      return Collections.emptyList();
+    }
+    
+    if (statutoryDescriptions == null) {
+      return descriptions;
+    }
+    
+    if (descriptions == null) {
+      return statutoryDescriptions;
+    }
+    
+    List<LocalizedValue> result = new ArrayList<>(descriptions);
+    
+    for (LocalizedValue statutoryDescription : statutoryDescriptions) {
+      if (StringUtils.isNotBlank(statutoryDescription.getValue())) {
+        int valueIndex = findLocalizedValueIndex(result, statutoryDescription.getType(), statutoryDescription.getLanguage());
+        if (valueIndex == -1) {
+          result.add(statutoryDescription);
+        } else {
+          LocalizedValue localizedValue = result.get(valueIndex);
+          localizedValue.setValue(String.format("%s%n%n%s", statutoryDescription.getValue(), localizedValue.getValue()));
+        }
+      }
+    }
+
+    return result;
+  }
+  
+  private int findLocalizedValueIndex(List<LocalizedValue> localizedValues, String type, String language) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused private "findLocalizedValueIndex" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)